master/internal/command/command.go

```diff
@@ -298,9 +298,6 @@ func (c *command) receiveSchedulerMsg(ctx *actor.Context) error {
 		c.userFiles = nil
 		c.additionalFiles = nil
 
-	case scheduler.ReleaseResources:
```

Comment:
We should let this message do no-op rather than delete this message otherwise a command would receive this message and panic (which falls on the "default" case in the switch statement.) due to `ErrUnexpectedMessage`

Reply:
I'd consider it a big bug if any scheduler *did* send this message though, because that means it's not taking into account the fact that this can't be preempted. For instance, if I had simply made terminate() a noop and left the fair share scheduler alone, it would have already allocated the slot for a preempted notebook to another task by the time it got that far. So one way or another, I'd like to make sure that we don't silently ignore the fact that this message is sent. Panicking is one route, although that does seem like an overreaction. Do we have any more graceful way to bubble up the fact that something is very wrong without being so dramatic?

Reply:
I see. That makes sense to me. thanks for the explanation!